isis/src/cassini/apps/cisscal/main.cpp

```diff
@@ -208,6 +205,10 @@ void IsisMain() {
                      _FILEINFO_);
   }
 
+  // we need to set output cube at the beginning of the program to check right
+  // away for CubeCustomization IsisPreference and throw an error, if necessary.
```

Comment:
This comment doesn't really make sense anymore. We're now making the cube at the end of the program.